ios/Podfile.lock

```diff
@@ -88,7 +88,7 @@ PODS:
     - GoogleUtilities/Logger
```

Comment:
why it is downgrading all libs?

Reply:
I can't say actually. I needed to reinstall Pods recently. That's what I got

Reply:
what pods version do you have?

Reply:
1.9.3

Reply:
strange, me too